### ISSUE TITLE
Offer users choice of roles on first visit to a network site

### ISSUE THREAD

**Issue by Ipstenu** [Ipstenu/join-my-multisite#3]
From [the forums](https://wordpress.org/support/topic/suggested-feature-offer-users-choice-of-roles-on-first-visit-to-a-network-site?replies=4#post-7130534)

> Admin can set in the plugin options that when the user visits a network site for the first time, the user can be offered a choice of roles. Then when the user visits that site for the first time, before he/she can do anything else he has to choose a role for himself in the per-site registration.

**Comment by avrcata** [comment]
Hi Mika,

Any news about this feature? I also use WPLMS where I have 2 types of accounts. Teacher and student.
This feature would be awesome!

**Comment by Ipstenu** [comment]
No, I need the sit-down-and-solve time for this one, and I just don't have a lot of free time for this at the moment :( It's staying open because I very much think I SHOULD add this.
